pypff-stubs/pypff.pyi

```diff
@@ -0,0 +1,193 @@
+import datetime
+from typing import Any, Iterator, Iterable, Optional
+"""
+The stubs file for pypff, a python wrapper for libpff
+
+We base our understanding of the structure of each class through reading the
+class definitions in [1]. One peculiarity of the pypff library are the large amount
+of unnecessary "getter" methods that fetch class instance variables. 
+
+[1] https://github.com/libyal/libpff/tree/main/pypff
+"""
+
+def file() -> File:
+    """
+    The constructor for th pypff.File object.
+
+    We cannot use __init__ to define this constructor, since the method call does not follow Python
+    naming convention
+    """
+
+class File:
+    ### Instance variables
+    # Unused instance variables
+    # size
+    # content_type
+    # encryption_type
+    # ascii_codepage
+    # root_item
+    # message_store
+    # name_to_id_map
+
+    """
+    The root folder.
+    """
```

Comment:
It seems like we shouldn't be using [docstrings for class attributes](https://stackoverflow.com/a/3051356)? The position of this docstring actually corresponds to the `class File` statement since it goes immediately under the definition. Maybe we should move this to the class docstring, or just use a normal comments.

Reply:
9ec1fd476cfdc42181cc4045ba978708caa720c7